app/views/install/compose.phtml

```diff
@@ -36,6 +36,11 @@ services:
   appwrite:
     image: <?php echo $organization; ?>/<?php echo $image; ?>:<?php echo $version."\n"; ?>
     container_name: appwrite
+    logging:
```

Comment:
Considering our docker compose file is already 500+ lines, we might wanna use some YML features to clean things up.

```yml
x-logging: &x-logging
    driver: 'json-file'
    options:
          max-file: 5
          max-size: 10m

services:
  appwrite:
    container_name: appwrite
    logging:
      <<: *x-logging
```
something like this should work

Reply:
For reference, I really wanna push for something like this => https://gist.github.com/TorstenDittmann/7dfd1c10345023066f1b98098426fb26

I might be creating a PR today :-)